File: Modul-4/Soal 3/tempCodeRunnerFile.py
def print_cross(zayne, rafa):
    if zayne > rafa:

        for i in range(zayne, rafa - 1, -1):
            print(f"{i} {zayne + rafa - i}", end="")
            if i > rafa:
                print(" - ", end="")
    else:
        for i in range(zayne, rafa + 1):
            print(f"{i} {rafa - i + zayne}", end="")
            if i < rafa:
                print(" - ", end="")
    print()

def main():
    caleb, sylus = map(int, input(" ").split())

    print_cross(caleb, sylus)
    
if __name__ == "__main__":
    main() 
 

File: Modul-5/Soal 1/tempCodeRunnerFile.c
#include <stdio.h>

int batas(int rafa, int sylus, int caleb, int zayne)

{
    int max = rafa;
    if (sylus > max) max = sylus;
    if (caleb > max) max = caleb;
    if (zayne > max) max = zayne;
    return max;
}
int main ()
{
    int rafa, sylus, caleb, zayne;
    scanf("%d %d %d %d", &rafa, &sylus, &caleb, &zayne);
    int hasil = batas (rafa, sylus, caleb, zayne);
    printf("%d", hasil);
    return 0;
}